### PR TITLE
Try to improve docs navigability somewhat

### PR DESCRIPTION
**Pull request by thomcc**
It's still not ideal, but this is an improvement. Let me know if you want me to explain why I felt any of the commits was worth doing, or if you'd prefer I remove it.

(I also don't mind removing myself from contributors if you'd rather, as this is trivial)

### CODE REVIEW

File: symphonia-utils-xiph/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Symphonia Xiph Utilities
 
+[![Docs](https://docs.rs/symphonia-xiph/badge.svg)](https://docs.rs/symphonia-xiph)
```
Comment (pdeljanov) [inline symphonia-utils-xiph/README.md:3]: symphonia-utils-xiph

File: symphonia-format-ogg/README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Symphonia OGG Demuxer
 
+[![Docs](https://docs.rs/symphonia-ogg/badge.svg)](https://docs.rs/symphonia-ogg)
```
Comment (pdeljanov) [inline symphonia-format-ogg/README.md:3]: `symphonia-format-ogg`

**Review by pdeljanov — CHANGES_REQUESTED** [review]
Couple of the badges are broken because the incorrect crate name is used.

**Comment by thomcc** [comment]
Oof!

**Review by pdeljanov — APPROVED** [review]

**Comment by pdeljanov** [comment]
Thanks!

**Pull request merged**